extensions/linear/src/components/CreateIssueForm.tsx

```diff
@@ -147,33 +147,38 @@ export default function CreateIssueForm(props: CreateIssueFormProps) {
             ...getCopyToastAction(copyToastAction, issue),
           };
 
-          if (values.attachments[0]) {
-            try {
-              await createAttachment({
-                issueId: issue.id,
-                url: values.attachments[0],
-              });
-            } catch (error) {
-              toast.style = Toast.Style.Failure;
-              toast.title = "Failed to create attachment";
-              toast.message = "The issue was still created, but the attachment could not be added.";
-            }
-          }
-
           reset({
             title: "",
             description: "",
             estimate: "",
             labelIds: [],
             dueDate: null,
             parentId: "",
+            attachments: [],
           });
 
-          if (hasMoreThanOneTeam) {
-            return focus(autofocusField);
-          }
+          hasMoreThanOneTeam ? focus(autofocusField) : focus("teamId");
```

Comment:
shouldn't this be the other way around?

Reply:
It's the right way because if you have more than one team, then you want to autofocus the field you've selected in the preferences (either the issue title or the team ID).

That being said, there's a mistake in the case where there is just one team. Since the teams dropdown is hidden, we want to focus the issue's title instead.